lib/pip.js

```diff
@@ -118,6 +118,12 @@ function installRequirements(
     }
     cmdOptions.push(dockerImage);
     cmdOptions.push(...pipCmd);
+
+    // If enabled slimming, strip out the caches, tests and dist-infos
+    if(options.slim == true){
```

Comment:
Can you make this a triple equal please? Javascript it weird, see here: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Equality_comparisons_and_sameness

Reply:
Oh, yeah! Thanks, changed it!